app/js/states/actions/actions.controller.js

```diff
@@ -55,6 +56,26 @@ function ActionsCtrl(
 
             $scope.featuredTopics = topics.slice(0, 3);
             $scope.extraTopics = topics.slice(3);
+
+            let product;
+            if (FilterService.getSelectedProduct() !== 'all') {
+                product = FilterService.getSelectedProduct();
+            }
+
+            Topic.get('incidents', product).success((topic) => {
+                let rulesWithHits = 0;
+
+                if (topic.rules) {
+                    topic.rules.forEach((rule) => {
+                        if (rule.hitCount > 0) {
+                            rulesWithHits++;
+                        }
+                    });
+                }
+
+                $scope.incidentCount = rulesWithHits;
```

Comment:
![01](https://user-images.githubusercontent.com/167352/27229540-40e41baa-52ac-11e7-8689-0da55933620f.png)

I observe this when selecting a group. Notice 2 incidents but 0 systems. The logic above looks right though - might be a glitch in the topics API actually